contrib/completion/bash/docker

```diff
@@ -169,6 +169,22 @@ __docker_capabilities() {
 	" -- "$cur" ) )
 }
 
+# a selection of the available signals that is most likely of interest in the
+# context of docker containers.
+__docker_signals() {
+	COMPREPLY=( $( compgen -W "
+		CONT
+		HUP
+		INT
+		KILL
+		QUIT
+		STOP
+		TERM
+		USR1
+		USR2
```

Comment:
I was actually unsure whether or not to include these variants and finally opted for the shorter list of completions. You are probably right that the alternatives should be added. Users would expect signal names to begin with _SIG_, so they would start typing _S_ and get unexpected results.
The prefixed variants should also be added for consistency reasons with `docker-compose`'s completion.